package.json

```diff
@@ -133,6 +133,16 @@
             false
           ]
         },
+        "deno.cacheOnSave": {
```

Comment:
Would you be able to update README.md as well? There is an outline of all the config there.